### ISSUE TITLE
Riv 4 appliance install w/Ubuntu Focal Fossa gets hpklinux and libssl1.1 errors

### ISSUE THREAD

**Issue by druidlabs** [ElvishArtisan/rivendell#836]
I downloaded the current Ubuntu Studio 20.04 ISO, installed it, brought it up to date on maintenance, then installed Rivendell using the instructions for a standalone appliance install on Focal Fossa.  The install failed with:
```
The following packages have unmet dependencies:
rivendell : Depends: hpklinux (>= 4.20.35) but it is not going to be installed
            Depends: libssl1.1 (>= 1.1.0) but it is not installable
```
Installing hpklinux manually gives: 
```
hpklinux : Depends: linux-headers-5.8.0-53-lowlatency but it is not installable
```
Trying to install the libssl and linux-headers packages fails with "is not available".

I note that the install instructions differ from the current Ubuntu Studio screens and menus.  Did Ubuntu sneak in some unexpected changes, or did I miss something?  Thanks.

**Comment by brettpatram** [comment]
I was running into a similar dependency issue on 22.04
There was no issue when I initially installed a test system approx 15 days ago from now
Around the time you reported your issue; Upon rebuilding another test system I was getting similar dependency errors for those two packages. 

A few days ago, I built up another test system and I am no longer getting these errors on 22.04 during installation.  It appears to have been fixed externally. 

If you perform package updates I wonder if "it" also has been fixed in similar way under 20.04?
